www-client/uget/uget-2.2.3.ebuild

```diff
@@ -0,0 +1,67 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="6"
+
+inherit gnome2-utils xdg-utils
```

Comment:
Usage of *gnome2-utils* for `icon_cache_update` has been deprecated a long time ago. I'd suggest you to just inherit *xdg.eclass* here, it'll help you a lot.